### ISSUE TITLE
Updated Dǝrsāna Mikāʾel and two magic works in BAVet121

### ISSUE THREAD

**Issue by MassimoVilla** [SChAth/ScAthiop#2673]
Dear all,
I would like to open a branch with the following aims:

- update LIT1295Dersan (Dǝrsāna Mikāʾel), adding incipits from BAVet82 and further information on the distribution of the textparts, which vary from MS to MS (e.g. BAVet82 and BAVet121)
- correct a small piece of information in LIT2835RepCh119
- create a work record for "ʾAsmāt-names of St Michael" as found in BAVet121. Incipit: በስመ፡ አብ፡ ... ኅቡዓት፡ አስማቲሁ፡ ለቅዱስ፡ ሚካኤል፡ ሊቀ፤ መላእክት፡ ኤኮስ፡ አስሌ፤ ኤፓ፤ ኤፓስ፡ ኤንካ፡ ኤንኪ፤
- create a work record for "ʾAsmāt-names of God found by St Michael", as found in BAVet121. Incipit: ያኮስ፡ አሜእ፡ ሌኬ፡ ኤንካዕ፡ ከዜዕ፤ ኮርናዕ፤ ክዮን፤ ኤፔካ፡ ኤሎ፤.

Shall I proceed?

**Comment by eu-genia** [comment]
This would be wonderful, just note that DM is a known challenge, there are many issues already 
https://github.com/BetaMasaheft/Documentation/issues/2361 and so far noone has had the energy to dedicate themselves to this task .

The plan was to create individual records for miracles, in which case the varying distribution of textparts will be resolved by the correct reference in msItems. (@CarstenHoffmannMarburg had offered himself and asked to be assigned but I am not sure whether he has done anything yet, I do not see any pull requests associated with issue #2361 )

If you do manage to work on `LIT1295Dersan `and also consider trying to disambiguate to `LIT4155MiraclesMichael `that would be highly appreciated.

**Comment by nafisa-valieva** [comment]
Dear Massimo,
I have a question on what exactly do you want to do on LIT2835RepCh119? To my knowledge, @eu-genia was planning to update records based on https://malkeagubae.com/works/2835/, but maybe you can do it yourself. You can have a look at what we decided to do in LIT2892RepCh173

**Comment by MassimoVilla** [comment]
Thank you all!
@eu-genia I 'm happy to do some sorting, creating separate records for miracles and monthly homilies (if there aren't any already), adding incipits/explicits, and associating them with the divs already existing in LIT1295Dersan. It is possible that more work will be needed in the future based on the features of other DM manuscripts, but I am happy to bring some progress now. I do not exclude that it may be necessary to create multiple DM records in addition to LIT1295Dersan, including a general record.

@CarstenHoffmannMarburg If you have advice (now or after PR) based on your experience, I'm all ears.

@nafisa-valieva in reality I only wanted to correct VATet82 into BAVet82 inside `<listWit/>`. If you want me to update the record based on LIT2892RepCh173, please tell me (but consider that I do not have much information from Grébaut-Tisserant)

**Comment by eu-genia** [comment]
I am doing the mapping, there is a separate issue https://github.com/BetaMasaheft/Documentation/issues/2640
@nafisa-valieva please leave the things apart, it is no problem at all to update the records, I have no hidden branches )

**Comment by CarstenHoffmannMarburg** [comment]
I must honestly admit that I lost sight of the task. I'm happy to do it now or later. But we should avoid working in parallel.
@MassimoVilla do you have a reason why you want to choose BAVet82 and have you compared it with BMLacq776? Which account do you prefer to update LIT1295Dersan? Do you want me to do it or do you want to do it yourself?
For the other three points (nos. 2 to 4) that you suggested, I cannot contribute because you have the knowledge and the texts for them.

**Comment by MassimoVilla** [comment]
@CarstenHoffmannMarburg we definitely should avoid working in parallel, I agree.
I chose BAVet82 only because it was more complete and detailed than BAVet121 (which I am working on), but I did not know that there were already other open issues with a better candidate for `edition`. If you have already received the task assignment and have started thinking about how to do it, we can proceed as follows: I update LIT1295Dersan with some comments about the distribution of single texts in the monthly readings, which is very different between BAVet82 and BAVet121 and might reflect two distinct and established transmittional traditions if confirmed by other MSS; then, once my branch is merged (together with the 2 new work records), I leave it to you to update LIT1295Dersan in a more systematic way. 
Could this work?

**Comment by CarstenHoffmannMarburg** [comment]
> we can proceed as follows: I update LIT1295Dersan with some comments about the distribution of single texts in the monthly readings, which is very different between BAVet82 and BAVet121 and might reflect two distinct and established transmittional traditions if confirmed by other MSS; then, once my branch is merged (together with the 2 new work records), I leave it to you to update LIT1295Dersan in a more systematic way.

@MassimoVilla, I think, you are done with your part in https://github.com/BetaMasaheft/Works/pull/1293. So, I will now go ahead and create a new branch.

**Comment by MassimoVilla** [comment]
Yes, thank you. The respective branch is merged and this issue can be closed
